### ISSUE TITLE
Property 'output' has conflicting type annotations

### ISSUE THREAD

**Issue by andycandy-de** [anatawa12/compile-time-constant#1]
Nice plugin =) It' s simple and it does exaclty what I want.

Clould you please resolve following gradle warning:
```
$ ./gradlew build --warning-mode all

> Task :createCompileTimeConstant UP-TO-DATE
Property 'output' has conflicting type annotations declared: @OutputFile, @Input; assuming @OutputFile. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0. See https://docs.gradle.org/6.5.1/userguide/more_about_tasks.html#sec:up_to_date_checks for more details.
```

All the best

**Comment by anatawa12** [comment]
Thank you for your reporting and I'm sorry for I have not noticed for long time.

**Comment by anatawa12** [comment]
Fixed in 332f25822977f934a50e07c447973c4183ee8b3b and released '1.0.2' onto mavenCentral.
